### ISSUE TITLE
I0425 10:13:21.205826  9167 solver.cpp:208] Iteration 220, lr = 0.01 I0425 10:13:21.206111  9167 solver.cpp:65] Iteration 220, loss = nan

### ISSUE THREAD

**Issue by lixyresearch** [BVLC/caffe#361]
I0425 10:12:49.611122  9167 solver.cpp:65] Iteration 180, loss = 6.55607
I0425 10:13:04.672341  9167 solver.cpp:208] Iteration 200, lr = 0.01
I0425 10:13:04.672637  9167 solver.cpp:65] Iteration 200, loss = 6.55622
I0425 10:13:04.978760  9167 solver.cpp:130] Snapshotting to caffe_dishll_train_iter_200
I0425 10:13:05.657246  9167 solver.cpp:137] Snapshotting solver state to caffe_dishll_train_iter_200.solverstate
I0425 10:13:21.205826  9167 solver.cpp:208] Iteration 220, lr = 0.01
I0425 10:13:21.206111  9167 solver.cpp:65] Iteration 220, loss = nan
